caikit/runtime/servicers/model_runtime_servicer.py

```diff
@@ -57,12 +57,14 @@ def loadModel(self, request, context):
             )
             caikit_config = get_config()
             if caikit_config.runtime.use_abortable_threads:
+                aborter = CallAborter(context)
```

Comment:
This aborter is now used to abort the top-level action as well as the internal thread doing the loading